cryostatio/cryostat-web
# feat(settings): categorize user settings

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Fixes: #781 
Depends on #789 

## Description of the change:

- [x] Categorize user settings (similar to OCP console). Settings can be ordered based using an order index.
- [x] Enable flip and add height constraints for setting menus.
- [x] Fix nested forms, anchors warnings.
- [ ] Tests for setting page.

## Motivation for the change:

See #781 

## How to manually test:
1. Run `CRYOSTAT_IMAGE=quay.io... sh smoketest.sh..`.
2. Go to setting page to see new changes.

## Screenshots
![new-setting-page](https://user-images.githubusercontent.com/68053619/211077579-d896f2be-4c56-423c-87e6-000514a6f2f5.png)
